libs/platform/src/lib/icon-tab-bar/components/icon-tab-bar-base.class.ts

```diff
@@ -159,6 +172,14 @@ export abstract class IconTabBarBase implements OnInit, OnChanges, OnDestroy {
         this.selected.emit(selectedItem)
     }
 
+    /** @hidden */
+    _keyHandlerHandler(event: any, tab: IconTabBarItem): void {
```

Comment:
_keyDownHandler

Reply:
why `any`, can we use KeyboardEvent?

Reply:
@dimamarksman Because I copy/paste this handler from the other component and didn't notice that it uses any.) Fixed